.travis.yml

```diff
@@ -21,6 +21,9 @@ stages:
     if: branch != rel/stable AND (branch =~ /^rel\//) AND type != pull_request
 
 jobs:
+  allow_failures:
+    - name: External ARM64 Build
+    - name: External ARM64 Deploy
```

Comment:
Presumably we want to remove this in the future?

Reply:
Once we feel that it's stable enough; I don't want the external build to break travis for now.